Library/Address.cs

```diff
@@ -85,17 +85,28 @@ internal override void WriteXml(XmlTextWriter xmlWriter)
         {
             xmlWriter.WriteStartElement("address");
 
-            xmlWriter.WriteElementString("first_name", FirstName);
-            xmlWriter.WriteElementString("last_name", LastName);
-            xmlWriter.WriteElementString("name_on_account", NameOnAccount);
-            xmlWriter.WriteElementString("company", Company);
-            xmlWriter.WriteElementString("address1", Address1);
-            xmlWriter.WriteElementString("address2", Address2);
-            xmlWriter.WriteElementString("city", City);
-            xmlWriter.WriteElementString("state", State);
-            xmlWriter.WriteElementString("zip", Zip);
-            xmlWriter.WriteElementString("country", Country);
-            xmlWriter.WriteElementString("phone", Phone);
+            if (FirstName != null)
```

Comment:
Yes, it would need to be set to an empty string to null out the fields. This might be considered a breaking change, but suppose the following situation:

You have address data on the server, but let's suppose it's missing a phone number. So instead of pulling the existing address data, you just set the phone number and then make the API call. The client will null out all the other values without you explicitly setting them.

This change would allow people to do this in a safer way, without necessarily needing to fetch from the API first before making changes.

We can put in the upgrade notes that they will need to change explicit setting of `null` to `""` in order to clear out values.

Reply:
> You have address data on the server, but let's suppose it's missing a phone number. So instead of pulling the existing address data, you just set the phone number and then make the API call. The client will null out all the other values without you explicitly setting them.

Can you explain what you mean by that? Perhaps with code? I'm not seeing where the problem is.

Reply:
Let's suppose you created an address like this:
```c#
var account = new Account("account-code")
{
    Address = new Address()
    {
        FirstName = "Benjamin",
        LastName = "Du Monde",
        Address1 = "123 main st",
        City = "New Orleans",
        State = "LA",
        Zip = "70129",
        Country = "US"
    }
};
account.Create();
```

Then later, you need to add the phone number. Since you already know the account code, you don't want to have to fetch the whole account object from the server. So you just create a new one, add the address information you want, and then update it.

```c#
var account = new Account("account-code")
{
    Address = new Address()
    {
        Phone = "555-555-5555"
    }
};
account.Update();
```
When you do this, it sends the following XML:
```xml
<?xml version="1.0" encoding="utf-8"?>
<account>
  <account_code>account-code</account_code>
  <address>
    <first_name />
    <last_name />
    <name_on_account />
    <company />
    <address1 />
    <address2 />
    <city />
    <state />
    <zip />
    <country />
    <phone>555-555-5555</phone>
  </address>
</account>
```
And now, everything else you created previously is cleared out. To prevent this, you could fetch the account first:
```c#
var account = Accounts.Get("account-code");
account.Address.Phone = "555-555-5555";
account.Update();
```
but if you do that, you have to make an extra (potentially unnecessary) network call, just to avoid having the Address object write a bunch of XML tags that have nothing to do with what you explicitly set. All you set was the `Phone`. This PR makes it such that all that is sent to the API is:
```xml
<?xml version="1.0" encoding="utf-8"?>
<account>
  <account_code>account-code</account_code>
  <address>
    <phone>555-555-5555</phone>
  </address>
</account>
```